tests/components/unifi/conftest.py

```diff
@@ -169,10 +170,19 @@ def fixture_request(
     def __mock_requests(host: str = DEFAULT_HOST, site_id: str = DEFAULT_SITE) -> None:
         url = f"https://{host}:{DEFAULT_PORT}"
 
-        def mock_get_request(path: str, payload: list[dict[str, Any]]) -> None:
+        def mock_get_request(
+            path: str, payload: list[dict[str, Any]], append_json: bool = True
+        ) -> None:
+            # APIV2 request respoonses have `meta` and `data` automatically appended
+            json = {}
+            if append_json:
+                json = {"meta": {"rc": "OK"}, "data": payload}
+            else:
+                json = payload
```

Comment:
We can skip the append json addition if we just validate the path
```suggestion
            if not path.startswith("/v2"):
                json = {"meta": {"rc": "OK"}, "data": payload}
            else:
                json = payload
```